code/datums/gamemode/factions/cult.dm

```diff
@@ -0,0 +1,54 @@
+/datum/faction/cult
+	ID = CULT
+	var/eldergod
+	var/deity_name
```

Comment:
so this var is called "deity_name" but it holds their title instead of their name?

Reply:
It can be either or any.